new_course/1_programming_fundementals/basic_enumerable_methods.md

```diff
@@ -1,51 +1,343 @@
-# Basic Enumerable Methods
-* what enumerable methods are
-* some can be used on hashes, others turn a hash into an array
+## Introduction
+
+The Enumerable module is a set of methods that is included in certain Ruby classes (like Array and Hash). You can even include it in your own classes, which will be covered later on. These methods are handy for looping through iterable objects in a collection (most commonly arrays, hashes, and ranges). They provide
```

Comment:
Modules and classes are something the student won't have been exposed to by this part of the course yet. Those will be the main topics in the section that comes after this section of lessons.

What would you think about changing the first line to something like this instead
```
Enumerables are a set of methods that are available on certain collections in ruby (like Array and Hash)
```